uniprotkb-common/src/main/java/org/uniprot/api/uniprotkb/common/service/groupby/GroupByService.java

```diff
@@ -49,11 +49,11 @@ public GroupByResult getGroupByResult(String query, String parentId) {
                 query);
     }
 
-    protected static boolean isTopLevelSearch(String parent) {
+    public static boolean isTopLevelSearch(String parent) {
         return StringUtils.isEmpty(parent);
     }
 
```

Comment:
Do we need this method static? If yes, we can simply use StringUtils.isEmpty in if condition. Not much code anyway